certified-connectors/Sociabble/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1073 @@
+                    "title": "Channels"
+                  },
+                  "description": "channelIds"
+                },
+                "audienceId": {
+                  "type": "string",
+                  "description": "The audience that can access this content",
+                  "title": "Audience"
+                },
+                "date": {
+                  "type": "string",
+                  "description": "The publication date of the content",
+                  "title": "Publication date"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Create an internal news",
+        "operationId": "CreateInternalNews",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "contents": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "title": {
+                        "type": "string",
+                        "description": "Write a title for your content",
+                        "title": "title",
+                        "x-ms-summary": "Title"
+                      },
+                      "summary": {
+                        "type": "string",
+                        "description": "Write a short summary of the content",
+                        "title": "summary",
+                        "x-ms-summary": "Summary"
+                      },
+                      "description": {
+                        "type": "string",
+                        "description": "Write your content",
+                        "title": "",
+                        "x-ms-summary": "Content"
+                      },
+                      "language": {
+                        "type": "string",
+                        "description": "Choose a language for the content",
+                        "title": "language",
+                        "x-ms-summary": "Language"
+                      },
+                      "isDefault": {
+                        "type": "boolean",
+                        "description": "If you want this language to be the default, choose yes",
+                        "title": "is the language default ?",
+                        "x-ms-summary": "Is default language"
+                      },
+                      "mediaIds": {
+                        "type": "array",
+                        "items": {
+                          "type": "string",
+                          "description": "The unique identifier from a previous media upload to use"
+                        },
+                        "title": "image",
+                        "description": ""
```

Comment:
@mdujardibs Please provide valid description value.

Documentation -[Define operation and parameter summaries and descriptions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
Done